biohackrxiv-paper-2022/paper-20/paper.md

```diff
@@ -0,0 +1,196 @@
+| 12 | Deployment Method         	| Means which technology is used to put in production different components                                                                                                                                                                                  	| -    	|        	|
+| 13 | Helpdesk Management System	| Main tool for user/helpdesk communication                                                                                                                                                                                                                 	| N    	|        	|
+| 14 | Helpdesk Portal Technology	| Federated EGA node helpe desk will provide  means of interacting with submitter, registering DPA, establishing DACs  and monitoring submissions.                                                                                                             	| N    	| 15     	|
+| 15 | Monitoring Tool           	| Means of providing an overview of the status of submissions. Good also for auditing purposes.                                                                                                                                                             	| N    	| 14     	|
+| 16 | Metadata Submission Method	| Means how the researchers will send the metadata to Central EGA through a cEGA submitter portal, node portal or another solution                                                                                                                          	| Y    	|        	|
+| 17 | Download Solution         	| Means for a Requester to access the data once approval has been granted.                                                                                                                                                                                  	| Y    	| 7, 8, 18   |
+| 18 | Data Access Tool          	| Means of enabling Requesters to Download the archived data. tool for facilitating DACs                                                                                                                                                                    	| N    	|        	|
+| 19 | General Contact           	| Email or website to contact the team deploying the solution                                                                                                                                                                                               	| Y    	|        	|
+
+Table: Federated EGA Technical Requirements
+
+This information is augumented with technologies utilized by current nodes (as of November 2022): in a fork of the [FEGA-Onboarding documentation](https://ahornos.github.io/FEGA-onboarding/topics/technical-operational/#technical-requirements).
+
+### Encryption
+
+All sensitive data uploaded to a FEGA node need to be encrypted using the crypt4gh standard[@crypt4gh]. For encrypting, apart from the file itself, all one needs is:
```

Comment:
Is crypt4gh supposed to be a link?

Reply:
no, there is a paper that is related to crypt4gh we should use that